client.go

```diff
@@ -214,11 +214,10 @@ func (c *client) unassignRoles(ctx context.Context, roleIDs []string) error {
 	var merr *multierror.Error
 
 	for _, id := range roleIDs {
-		var rawResponse *http.Response
-		ctxWithResp := policy.WithCaptureResponse(ctx, &rawResponse)
-		if _, err := c.provider.DeleteRoleAssignmentByID(ctxWithResp, id); err != nil {
-			// If a role was deleted manually then Azure returns a error and status 204
-			if rawResponse != nil && rawResponse.StatusCode == http.StatusNoContent || rawResponse.StatusCode == http.StatusNotFound {
+		if _, err := c.provider.DeleteRoleAssignmentByID(ctx, id); err != nil {
+			// If a role was deleted manually then Azure returns an error and status 204
```

Comment:
Does manually mean out-of-band, i.e. not by the plugin?

Reply:
That is correct. I updated the comment to reflect that it is an 'out-of-band' change to be more clear